cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/batch/distributed/MapReduceContainerLauncher.java

```diff
@@ -61,6 +62,9 @@ public static void launch(String mainClassName, String[] args) {
       throw new IllegalStateException("Failed to remove main class resource " + resource);
     }
 
+    // Install the JUL to SLF4J Bridge
+    SLF4JBridgeHandler.install();
```

Comment:
It is better to load this class from the MainClassLoader constructed below and use reflection to invoke the method